### ISSUE TITLE
A option to display the name of file

### ISSUE THREAD

**Issue by sang96011** [skaut/skaut-google-drive-gallery#1869]
I would like to have a option to display the file name.

**Comment by marekdedic** [comment]
Hi,
thanks for the suggestion. I have a few questions - why do you want to do this? Do you know that if you add a label to the image in Google Drive, it will be shown as a description? Maybe that would be a better fit for your use-case...

**Comment by sang96011** [comment]
I would like to have it so that it can be easily identified without having to play it (for videos).
Like this image below
![image](https://user-images.githubusercontent.com/13704700/225362681-2fc9d88f-f55c-4db5-aea6-43a3348c0c16.png)

**Comment by marekdedic** [comment]
Ok, wouldn't a big overlay/play sign be a better solution for that? As in #235?

**Comment by sang96011** [comment]
I think #235 is for distinguishing photo and video, but I would like more details, to identify one video with another video or photo.

**Comment by sang96011** [comment]
I got it. Thanks for the explanation.

**Comment by marekdedic** [comment]
Yeah, @kalich5 commented about the design and I agree. I am still trying to grasp what the use-case for this would be - maybe we can find an even better solution or maybe we figure out the problem is somewhere else...

You said you want to distinguish one video from another
1) Is the thumbnail not enough?
2) Would the file label be better that the filename?
3) Why do you want to do this? How do you plan on using the gallery?
4) Does this apply to photos as well?

Thanks

**Comment by sang96011** [comment]
Thank you for your hard work.
I will answer as below.

1. Is the thumbnail not enough?
-> Yes. For videos, The thumbnail is not enought information to distinguish.

2. Would the file label be better that the filename?
-> I'm not sure about this because i don't understand about "file label". I've researched it and i think it's for classification purpose.[Get started as a Drive labels admin](https://support.google.com/a/answer/9292382?hl=en) . 

3. Why do you want to do this? How do you plan on using the gallery?
-> As I said above. I want to distinguish one video from another. In my case, the videos are episodes of a movie.

4. Does this apply to photos as well?
-> I don't think it's necessary for photos. But if there is, even better.

Thanks.

**Comment by marekdedic** [comment]
Ok, thanks, that makes sense. I probably won't get to this issue in the near future, but I'll mark it as accepting PRs.

Oh and sorry, I mistakenly said "labels", it is actually called "descriptions" - you can add one for images in Google Drive. So it's something different from the link you posted
